core/modules/qana/CheckAggregation.h

```diff
@@ -63,7 +63,7 @@ class CheckAggregation {
      * @return void
      */
     inline void operator()(query::ValueExpr::FactorOp const& fo) {
-        if (!fo.factor.get());
+        if (!fo.factor.get()) {}
```

Comment:
It was added by @wangd in commit c6dcfc0bba09c109bf4af09fb305f3e9158fd309 but there wasn't an explanation as to what it was doing. I hope it wasn't relying on a side-effect of `get()`.